src/components/address-bar/container.tsx

```diff
@@ -14,13 +14,14 @@ export interface Properties extends PublicProperties {
   history: History;
   route: string;
   deepestVisitedRoute: string;
+  app: string;
```

Comment:
If I'm not mistaken the type should be `Apps`

Reply:
Thanks!